pvlib/clearsky.py

```diff
@@ -771,7 +777,7 @@ def rmse(alpha):
                       % max_iterations, RuntimeWarning)
 
     # be polite about returning the same type as was input
-    if isinstance(measured, pd.Series):
+    if ispandas:
```

Comment:
What's the advantage in declaring `ispandas` and only using it once? To be more idiomatic than `isinstance()`?

Reply:
the line introduced above, `measured = np.asarray(measured)`, would make this `isinstance` always evaluate `False`. So I moved the check above the reassignment. I also thought of doing something like `measured_ = np.asarray(measured)`. Modifying the check resulted in a smaller diff, though.

Reply:
I get it now. Thanks for clarifying that.